spec/requests/api/connect/v3/systems/products_controller_spec.rb

```diff
@@ -80,6 +80,20 @@
         end
       end
     end
+
+    describe 'activations' do
+      subject(:request) { post url, headers: headers, params: payload }
+
+      context 'when the product was not already activated on this system' do
```

Comment:
Is this the opposite of what you mean?

I think in this one you create an activation.

Reply:
oops! will fix thanks